### PR TITLE
Small improvements for page normalizer

### PR DESCRIPTION
**Pull request by tobiasdiez**
See test cases.

- [ ] Change in CHANGELOG.md described
- [x] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)

### CODE REVIEW

File: src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java [hunk 1]
```diff
@@ -36,6 +36,11 @@ public void ignoreWhitespaceInPageNumbers() {
     }
 
     @Test
+    public void removeWhitespaceForSinglePageNumber() {
+        expectCorrect("   1  ", "1");
```
Comment (stefan-kolb) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java]: Doesn't our Parser and Writer always trim fields?

File: src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java [hunk 2]
```diff
@@ -66,6 +71,16 @@ public void replacesLongDashWithDoubleDash() {
     }
 
     @Test
+    public void removesPagePrefix() {
+        expectCorrect("p.50", "50");
+    }
+
```
Comment (stefan-kolb) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java]: just remove not removes to be consistent?

File: src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java (hunk 1 above)
Comment (tobiasdiez) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java]: Yes they do. I added this test also to make sure that single pages are detected.

Comment (stefan-kolb) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java]: Yeah, keep it :+1:

File: src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java (hunk 2 above)
Comment (tobiasdiez) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatterTest.java]: Done

File: src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java [hunk 3]
```diff
@@ -7,6 +7,8 @@
 import net.sf.jabref.logic.formatter.Formatter;
 import net.sf.jabref.logic.l10n.Localization;
 
+import com.mysql.jdbc.StringUtils;
```
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java]: Do we really want to use stuff from mysql library here?

Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java]: Is com.google.common.base.Strings better?

Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java]: Good question :smile: I was just noticing this proprietary import that seemed like strongly mysql-related too me.

**Pull request merged**